messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/AddressSettingAdd.java

```diff
@@ -192,13 +161,11 @@ static AddressSettings createDefaultAddressSettings() {
         settings.setAutoCreateAddresses(AddressSettingDefinition.AUTO_CREATE_ADDRESSES.getDefaultValue().asBoolean());
         settings.setAutoDeleteAddresses(AddressSettingDefinition.AUTO_DELETE_ADDRESSES.getDefaultValue().asBoolean());
         settings.setAutoDeleteCreatedQueues(AddressSettingDefinition.AUTO_DELETE_CREATED_QUEUES.getDefaultValue().asBoolean());
-        settings.setMaxReadPageBytes(AddressSettingDefinition.MAX_READ_PAGE_BYTES.getDefaultValue().asInt());
         settings.setDeadLetterAddress(asSimpleString(DEAD_LETTER_ADDRESS.getDefaultValue(), null));
         settings.setExpiryAddress(asSimpleString(EXPIRY_ADDRESS.getDefaultValue(), null));
         settings.setExpiryDelay(AddressSettingDefinition.EXPIRY_DELAY.getDefaultValue().asLong());
         settings.setDefaultLastValueQueue(AddressSettingDefinition.LAST_VALUE_QUEUE.getDefaultValue().asBoolean());
         settings.setMaxDeliveryAttempts(AddressSettingDefinition.MAX_DELIVERY_ATTEMPTS.getDefaultValue().asInt());
-        settings.setMaxRedeliveryDelay(AddressSettingDefinition.MAX_REDELIVERY_DELAY.getDefaultValue().asLong());
```

Comment:
You should also remove the default value on the attribute definition

Reply:
I wanted to say that would require a schema bump but it looks like the default values are not in the schema to begin with. Still, the attribute does technically have a default value, it just isn't explicitly set, shouldn't we mention that in the description?

Reply:
Yes a schema bump is required since we need transformation of the default values

Reply:
You need also to add transformers

Reply:
I don't follow. A transformer is for changing old attributes into new attributes, right? Since we don't use the default value anywhere we don't care if the old attributes have it.

Reply:
Transformers are there to transform new configuration into old configuration and fail before operation is sent to old systems.

Reply:
I still don't understand, the default value being present is either a bug (if the default configuration relies on it) or it doesn't matter so adding it back for the old configuration would basically be supporting the buggy behavior.